### DISCUSSION TITLE
Suggested change to line 259 in function_4.py

### DISCUSSION THREAD

**Discussion by okinawaninja** [H3rz3n/davinci-helper#12] [AMD GPU Support]
Hi there, thank you for your amazing tool. it somehow cured my 4 hour plague of installing davinci resolve on linux. 
During this journey, I encountered the following error

> Traceback (most recent call last):
  File "/usr/lib/python3.13/site-packages/davinci_helper/functions/logic/function_4.py", line 593, in <module>
    gpu_supported_nvidia, gpu_supported_amd, gpu_supported_intel = start_gpu_support_search(gpu_vendor, gpu_model_name_list, gpu_model_number_list)
                                                                   ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/site-packages/davinci_helper/functions/logic/function_4.py", line 159, in start_gpu_support_search
    gpu_supported_amd = check_amd_gpu_support(gpu_model_name_list, gpu_model_number_list)
  File "/usr/lib/python3.13/site-packages/davinci_helper/functions/logic/function_4.py", line 259, in check_amd_gpu_support
    print(_(f"A compatible AMD GPU was found : {gpu_name_placeholder}").format(gpu_name_placeholder = gpu_name))
                                                ^^^^^^^^^^^^^^^^^^^^
NameError: name 'gpu_name_placeholder' is not defined

Replacing line 259 with `print(f"A compatible AMD GPU was found : {gpu_name}")
` fixed it for me, and allowed me to continue with the fourth (gpu driver check) step

**Comment by H3rz3n** [comment]
> Hi there, thank you for your amazing tool. it somehow cured my 4 hour plague of installing davinci resolve on linux. 
> During this journey, I encountered the following error
> 
> > Traceback (most recent call last):
>   File "/usr/lib/python3.13/site-packages/davinci_helper/functions/logic/function_4.py", line 593, in <module>
>     gpu_supported_nvidia, gpu_supported_amd, gpu_supported_intel = start_gpu_support_search(gpu_vendor, gpu_model_name_list, gpu_model_number_list)
>                                                                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/usr/lib/python3.13/site-packages/davinci_helper/functions/logic/function_4.py", line 159, in start_gpu_support_search
>     gpu_supported_amd = check_amd_gpu_support(gpu_model_name_list, gpu_model_number_list)
>   File "/usr/lib/python3.13/site-packages/davinci_helper/functions/logic/function_4.py", line 259, in check_amd_gpu_support
>     print(_(f"A compatible AMD GPU was found : {gpu_name_placeholder}").format(gpu_name_placeholder = gpu_name))
>                                                 ^^^^^^^^^^^^^^^^^^^^
> NameError: name 'gpu_name_placeholder' is not defined
> 
> Replacing line 259 with `print(f"A compatible AMD GPU was found : {gpu_name}")
> ` fixed it for me, and allowed me to continue with the fourth (gpu driver check) step

Thank you for pointing me that out, I will fix it in a couple of minutes right now 😄

The help of the community is fundamental to make this project better, thank you for using DaVinci Helper.
